CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 #### Fixes
 
+* [#1710](https://github.com/ruby-grape/grape/pull/1710): Make `declared` transform passed params to what is expected - [@pablonahuelgomez](https://github.com/pablonahuelgomez).
```

Comment:
This is a bit too vague. The bug being fixed is about nested objects or something like that, care to make it a bit more specific?

Reply:
Of course, tried to make it more specific.